### PR TITLE
Move to menpo 0.7.x

### PR DESCRIPTION
**Pull request by jabooth**

### CODE REVIEW

**Comment by jabooth** [comment]
Jenkins retest this please

**Comment by patricksnape** [comment]
@jabooth Check this please. PLY and OFF are missing and added as issues. This is a breaking change.

File: conda/bld.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
-robocopy %RECIPE_DIR%\.. . /E
+robocopy %RECIPE_DIR%\.. . /E /NFL /NDL
```
Comment (jabooth) [inline conda/bld.bat]: out of interest what's this? Needed for any other projects?

**Comment by jabooth** [comment]
@patricksnape code looks great, will test with STL shortly. No issue with loss of PLY for now.

**Comment by jabooth** [comment]
confirmed it's working great with STL's! LGTM :)

**Pull request merged**